joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/XTypeHintCallLinker.scala

```diff
@@ -60,7 +60,7 @@ abstract class XTypeHintCallLinker(cpg: Cpg) extends CpgPass(cpg) {
     }
   }
 
-  private def createMethodStub(methodName: String, call: Call, builder: DiffGraphBuilder): NewMethod = {
+  def createMethodStub(methodName: String, call: Call, builder: DiffGraphBuilder): NewMethod = {
```

Comment:
Same here, this is no longer overriden
```suggestion
  private def createMethodStub(methodName: String, call: Call, builder: DiffGraphBuilder): NewMethod = {
```

Reply:
Made it `protected` since it needs to be accessed by the derived class.